vault/resource_identity_oidc_provider.go

```diff
@@ -0,0 +1,169 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+const identityOIDCProviderPathPrefix = "identity/oidc/provider"
+
+func identityOIDCProviderResource() *schema.Resource {
+	return &schema.Resource{
+		Create: identityOIDCProviderCreateUpdate,
+		Update: identityOIDCProviderCreateUpdate,
+		Read:   identityOIDCProviderRead,
+		Delete: identityOIDCProviderDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Description: "The name of the provider.",
+				Required:    true,
+			},
+			"issuer": {
+				Type:     schema.TypeString,
+				ForceNew: true,
```

Comment:
Any reason for `ForceNew: true` here? This should be updatable without needing to delete/recreate the provider.

Reply:
I guess this probably isn't an "input" for creating this resource. If I understand correctly, it's replaced by `https_enabled` and `issuer_host`.

Reply:
You are correct, Austin! Thanks for clarifying that `issuer` fields can be updated on OIDC Providers. Removed the `ForceNew` behavior from this field in the latest commit